commands.md

```diff
@@ -8,6 +8,7 @@ Command | Implemented By | Who can run it | Description
 `/unassign [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | anyone | Unassigns specified people (or yourself if no one is specified). Target must already be assigned.
 `/cc [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | anyone | Request review from specified people (or yourself if no one is specified). Target must be a kubernetes org member.
 `/uncc [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | anyone | Dismiss review request for specified people (or yourself if no one is specified). Target must already have had a review requested.
+`/help` | prow [help](./prow/plugins/help) | anyone | If issue is unassigned, adds help-wanted label. Label is removed when issue is assigned.
```

Comment:
Got it. Sorry that I misread :) I just thought this was run on all open issues, like `needs-sig` label.